rct229/rulesets/ashrae9012019/ruleset_functions/get_primary_secondary_loops_dict.py

```diff
@@ -73,16 +73,12 @@ def get_primary_secondary_loops_dict(rmi_b):
         cfl_id = chilled_fluid_loop["id"]
         if cfl_id in chiller_loop_ids and cfl_id in non_process_chw_coil_loop_ids:
             # No loop in baseline shall be
-            tmp_primary_secondary_loops_dict = dict()
-            break
-        elif cfl_id in chiller_loop_ids:
-            if all(
-                child_loop["id"] in non_process_chw_coil_loop_ids
-                for child_loop in getattr_(
-                    chilled_fluid_loop, "FluidLoop", "child_loops"
-                )
-            ):
-                primary_loops.append(chilled_fluid_loop)
+            tmp_primary_secondary_loops_dict = {}
```

Comment:
You will have to break here.
So in this scenario,
```python
# reset the primary loop
primary_loop = []
break
```
You don't need to reset the `tmp_primary_secondary_loop_dict` here because it does nothing in this loop.